lib/importer.py

```diff
@@ -236,7 +236,7 @@ def linkToMyPlexAccount() -> MyPlexAccount:
         return None
 
     # show the user the pin
-    dialog.ok(localize(32015), localize(32053))
+    dialog.ok(localize(32015), localize(32053) + normalizeString(f"\n{pinLogin.pin}"))
```

Comment:
I'm not sure I understand this change?

Reply:
This is another one that needs to squashing.

The change was actually:
`dialog.ok(localise(32015), localise(32053), '[COLOR FFE5A00D]{}[/COLOR]'.format(pinLogin.pin))`

-> 

`dialog.ok(localize(32015), localize(32053) + normalizeString(f"\n{pinLogin.pin}"))`

It was just done in multiple steps through the different commits while I was testing. That's the "MyPlex Link Fix" from my original notes.

Reply:
Does `\n` work to show the pin on a new line? And now it's missing the `[COLOR FFE5A00D][/COLOR]`.

Reply:
Yeah, \n pushed it onto the next line. I'll try adding the color back in and throw some tabs on to try to get it more centered.

Reply:
Removed the newline and got the color back in. New line looked weird without it being centered and bigger or something.